ext/standard/basic_functions.c

```diff
@@ -1510,7 +1510,7 @@ PHPAPI int _php_error_log_ex(int opt_err, char *message, size_t message_len, cha
 
 		case 4: /* send to SAPI */
 			if (sapi_module.log_message) {
-				sapi_module.log_message(message, -1);
+				sapi_module.log_message((char *) message, -1);
```

Comment:
Adjust the `log_message` API to accept const char as well? I'm okay with doing this in a followup though.

Reply:
many API  arguments of SAPI missing the const tag, just a little trouble, but I will complete it soon